doc/releasing.md

```diff
@@ -48,6 +61,18 @@ Do the following when releasing:
         +silk-create-asset-group \
         --branch <branch>
      ```
+   - Create a new Snyk reference target. Track the newly created release branch. Copy the organization ID from [Snyk settings](https://app.snyk.io/org/dev-prod/manage/settings). Run `cmake` first so generated source files are present. Example for `rx.y`:
+     ```bash
+     cmake -S. -Bcmake-build -D BUILD_TESTING=OFF
+     cmake --build cmake-build --target mongocrypt
+     snyk auth
+     snyk monitor \
```

Comment:
May want to precede this command with [`snyk test`](https://docs.snyk.io/snyk-cli/commands/test) + `--print-dep-paths` to validate 3PD detection. e.g. a local test produces:
```
Dependencies:


  https://github.com|mongodb/mongo-c-driver@1.17.7
  purl: pkg:generic/mongo-c-driver@1.17.7?download_url=https%3A%2F%2Fgithub.com%2Fmongodb%2Fmongo-c-driver%2Farchive%2F1.17.7.tar.gz
  confidence: 1.000
  matching files:
    - local-build/_deps/embedded_mcd-src/src/common/common-b64-private.h
    - local-build/_deps/embedded_mcd-src/src/common/common-b64.c
    - local-build/_deps/embedded_mcd-src/src/common/common-md5-private.h
    ... and 367 more files
```

Admittedly it seems like there's little we can do atm to improve Snyk's 3PD detection even after the CMake generation stage (no IntelDFP found?).

Reply:
Update to add step with `--print-dep-paths` to verify libbson is found.

> it seems like there's little we can do atm to improve Snyk's 3PD detection even after the CMake generation stage (e.g. no IntelDFP found).

[Troubleshooting C/C++](https://docs.snyk.io/getting-started/supported-languages-and-frameworks/c-c++/troubleshooting-c-c++ notes):

> If your scan does not find any dependencies submit a request to support.

I [filed a request](https://support.snyk.io/hc/en-us/requests/new) to add IntelDFP, and am awaiting response.